File: app/src/main/java/com/taxi/friend/taxifriendclient/models/DriverLocation.java
package com.taxi.friend.taxifriendclient.models;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonInclude;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@JsonIgnoreProperties(ignoreUnknown = true)
@JsonInclude(JsonInclude.Include.NON_NULL)
@Builder
@AllArgsConstructor
@Getter
@Setter
@NoArgsConstructor
public class DriverLocation {
    private String id;
    private String name;
    private Double Latitude;
    private double longitude;
    private String status;
    private int direction;
}

File: app/src/main/java/com/taxi/friend/taxifriendclient/models/DriverLocation.kt
package com.taxi.friend.taxifriendclient.models

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonInclude


@JsonIgnoreProperties(ignoreUnknown = true)
@JsonInclude(JsonInclude.Include.USE_DEFAULTS)
class DriverLocation( id: String = "", name: String = "",  latitude: Double = 0.0, longitude: Double = 0.0, status: String = "", direction: Int = 0) {
    val id = id
    val name = name
    val latitude = latitude
    val longitude = longitude
    val status = status
    val direction = direction
}
